src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverMetrics.java

```diff
@@ -77,11 +77,11 @@ public class ResourceResolverMetrics {
     private ServiceRegistration<Gauge<Long>> numberOfVanityPathBloomFalsePositiveGauge;
     private Supplier<Long> numberOfVanityPathBloomFalsePositiveSupplier = ZERO_SUPPLIER;
 
-    // number of aliases
-    private ServiceRegistration<Gauge<Long>> numberOfAliasesGauge;
-    private Supplier<Long> numberOfAliasesSupplier = ZERO_SUPPLIER;
+    // number of resources with aliased children
+    private ServiceRegistration<Gauge<Long>> numberOfResourcesWithAliasedChildrenGauge;
+    private Supplier<Long> numberOfResourcesWithAliasedChildrenSupplier = ZERO_SUPPLIER;
```

Comment:
Not really this PR, just related: I think all `Supplier` fields need to be volatile. It looks like they can be updated at any moment. Am I missing something?

Reply:
Maybe. In practive it's only set once in MapEntries's init code (not it's a supplier, not the actual value).